### PR TITLE
Passing real Go types into Cassandra driver.

### PR DESCRIPTION
**Pull request by codingconcepts**

### CODE REVIEW

File: lib/output/cassandra.go [hunk 1]
```diff
@@ -382,6 +342,50 @@ func (c *cassandraWriter) writeBatch(session *gocql.Session, msg types.Message)
...
+
+		values := make([]interface{}, 0, len(args))
+		for _, arg := range args {
+			values = append(values, stringValue(arg.String(0, msg)))
+		}
+
+		return values, nil
+	}
+
+	argsMapping, err := bloblang.NewMapping("", c.conf.ArgsMapping)
```
Comment (Jeffail) [inline lib/output/cassandra.go:365]: It's possible that this parsing is actually quite expensive so we probably ought to do it only once in the constructor, and the same goes for the args fields. It's not as tidy but it also means we'd capture mapping errors earlier.

**Review by Jeffail — COMMENTED** [review]
Thanks @codingconcepts, looks good but left one comment. I'm also going to mark the old `args` field as deprecated and remove it from the docs as the new mapping field is cooler, but I'll take care of that after merging.

**Comment by codingconcepts** [comment]
Hey @Jeffail, great shout. That's done 🙂

**Comment by Jeffail** [comment]
Awesome, looks great, thanks @codingconcepts!

**Pull request merged**